jetty-core/jetty-start/src/main/java/org/eclipse/jetty/start/Environment.java

```diff
@@ -23,6 +23,11 @@
 import java.util.List;
 import java.util.Properties;
 
+/**
+ * A start environment that contains the configurations that will be used to
+ * build a runtime {@code org.eclipse.jetty.util.component.Environment} via
+ * {@code --env} arguments passed to {@code org.eclipse.jetty.xml.XmlConfiguration#main(java.lang.String...)}
+ */
 public class Environment
```

Comment:
Rename to something different from `Environment` to not confuse it with the `jetty-util` Environment?
Perhaps `Settings`?

Reply:
@joakime no because they are at different points in their lifecycle.  In start and env is properties, xmls and libs, whilst in the runtime an env is a classloader and attributes.